src/traffic_server/InkAPI.cc

```diff
@@ -7801,367 +7801,412 @@ TSSkipRemappingSet(TSHttpTxn txnp, int flag)
 
 template <typename T>
 inline void *
-_memberp_to_generic(T *ptr, std::type_info const *&type)
-{
-  type = &typeid(T);
+_memberp_to_generic(T *ptr, MgmtConverter const *&conv)
```

Comment:
Rather than just type information, return a converter to manage the interaction. To avoid excess code and updates, this does the type information locally and if it's one of the few types that were checked, provide a generic identity converter.